src/main/java/com/monumental/models/Role.java

```diff
@@ -0,0 +1,7 @@
+package com.monumental.models;
+
+public enum Role {
```

Comment:
Can we make this `static`?

Reply:
It's basically a class so making it static doesn't really do anything. You still access it like `Role.RESEARCHER`